melodies_monet/plots/surfplots.py

```diff
@@ -991,6 +1003,7 @@ def make_boxplot(comb_bx, label_bx, ylabel = None, vmin = None, vmax = None, out
     savefig(outname + '.png', loc=4, logo_height=100)
   
 def make_multi_boxplot(comb_bx, label_bx,region_bx,region_list = None, model_name_list=None,ylabel = None, vmin = None, vmax = None, outname='plot',  
+>>>>>>> origin/develop
```

Comment:
What does this line do? It looks like it may not need to be here.

Reply:
yeah. this is what we do to track change. I have nothing to do here. I will delete it

Reply:
deleted it.